validator-network/src/network_impl.rs

```diff
@@ -305,9 +302,12 @@ where
         secret_key: &SecretKey,
     ) -> Result<(), Self::Error> {
         let peer_id = self.network.get_local_peer_id();
-        let record = ValidatorRecord::new(peer_id);
+        let record = ValidatorRecord::new(
+            peer_id,
+            OffsetDateTime::now_utc().unix_timestamp() as u64 * 1000,
```

Comment:
```suggestion
            OffsetDateTime::now_utc().unix_timestamp_nanos() / 1_000_000 as u64,
```
Is there a reason for choosing milliseconds in particular?

Reply:
Left it as it is since `nanos` returns a `i128` and I believe it's more natural to convert from a `i64`. No specific reason to use millis, I used the same timestamp resolution that we use in blocks.

Reply:
Converting from seconds means the lowest three digits will always be 000. It seems it's more correct to get it from nanos so that the lowest three digits actually carry information.

Reply:
Ah yes, makes sense. Changed it to use `nanos`